pyInterface/bindings/storageFormats/eventMetadata_py.cc

```diff
@@ -86,6 +86,9 @@ void rpwa::py::exportEventMetadata() {
 		.def("productionKinematicsParticleNames", &eventMetadata_productionKinematicsParticleNames)
 		.def("decayKinematicsParticleNames", &eventMetadata_decayKinematicsParticleNames)
 		.def("additionalSavedVariableLables", &eventMetadata_additionalSavedVariableLables)
+		.def("getAdditionalSavedValue", &rpwa::eventMetadata::getAdditionalSavedValue)
```

Comment:
In ROOTPWA we usually not decorate getters with "get", so it would be nice to rename "getAdditionalSavedValue" -> "additionalSavedValue"